dChatServer/PlayerContainer.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 #include <map>
 #include "dCommonVars.h"
+#include "eGameMasterLevel.h"
```

Comment:
still including the header, if you are going the static_cast route

Reply:
whoops. fixing...